### PR TITLE
Upgrade to v3.32.0 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1396
Fixes: #1395
Fixes: #1392

Also add the ability to specify `aws.cloudwatch` `LogResourcePolicy` as iam.PolicyDocument

### CODE REVIEW

File: sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs [hunk 1]
```diff
@@ -13,16 +13,16 @@ namespace Pulumi.Aws.AppAutoScaling.Inputs
...
-        public Input<int>? MaxCapacity { get; set; }
+        public Input<string>? MaxCapacity { get; set; }
 
         /// <summary>
-        /// The minimum capacity.
+        /// The minimum capacity. At least one of `min_capacity` or `max_capacity` must be set.
         /// </summary>
         [Input("minCapacity")]
-        public Input<int>? MinCapacity { get; set; }
+        public Input<string>? MinCapacity { get; set; }
```
Comment (mikhailshilkov) [inline sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs:25]: This is breaking, let's triage

Comment (stack72) [reply sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs:25]: So they changed this to allow the equivalent of nullable :/ 

Do we want to accept that or do we want to try and back that out?

Comment (stack72) [reply sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs]: @mikhailshilkov I just updated this as per the old workings

Comment (mikhailshilkov) [reply sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs]: But... now it is a `bool` somehow?

Comment (stack72) [reply sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs]: *NOW* it's fixed :) That was my derp!

**Review by mikhailshilkov — APPROVED** [review]

**Pull request merged**